test/framework/integration_framework/integration_test_framework.hpp

```diff
@@ -57,14 +57,14 @@ namespace integration_framework {
    public:
     /**
      * Construct test framework instance
-     * @param maximum_proposal_size - (default = 10) Maximum amount of
-     * transactions per proposal
+     * @param maximum_proposal_size - Maximum amount of transactions per
```

Comment:
Since transaction is a countable noun, correct form will be "maximum **number** of transactions"